graphql/mutation_resolver.go

```diff
@@ -1219,6 +1217,26 @@ func (r *mutationResolver) DeleteSubscriptions(ctx context.Context, subscription
 	return len(subscriptionIds), nil
 }
 
+// RemoveFavoriteProject is the resolver for the removeFavoriteProject field.
+func (r *mutationResolver) RemoveFavoriteProject(ctx context.Context, identifier string) (*restModel.APIProjectRef, error) {
+	p, err := model.FindBranchProjectRef(identifier)
```

Comment:
I know you just moved these resolvers, but out of curiosity why do these return the project as opposed to just like, Okay? Is the returned value used for anything? Seems incorrect to return the partial branch project as opposed to the merged project (but i also recognize merging it is a waste when its not even used)

Reply:
Yeah, this is kind of a result of us not following good mutation design, we should really be returning a payload object that we can freely edit the fields of. I think it's something we eventually want to change but I think we have to think through how we can change the mutation return values in a backwards compatible way.